runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -370,6 +370,7 @@ std::shared_ptr<exec::ExecutorMap> Compiler::compile(void)
   {
     _subgraphs->iterate([&](const ir::SubgraphIndex &index, ir::Graph &subg) {
       executors->emplace(index, std::make_unique<interp::InterpExecutor>(subg));
+      subg.setSubgraphs(_subgraphs);
```

Comment:
Please find line 397
```
subg.setSubgraphs(nullptr);
```

runtime/onert/core/src/interp/InterpExecutor.test.cc
```
auto subgs = std::make_shared<onert::ir::Subgraphs>();
subgs->push(onert::ir::SubgraphIndex{0}, _graph);
_graph->setSubgraphs(subgs);
```

Other usages indicates the `_graph` should get reference to `_subgraphs`. Without this, the reference count is zero and it seemingly causes memory dangling.